### PR TITLE
split long attributes

### PR DESCRIPTION
**Pull request by yecril71pl**
Allow the user to edit the HTML file in a normal text editor.

### CODE REVIEW

File: src/shell.html [hunk 1]
```diff
@@ -190,12 +190,1028 @@
...
+     d="m
```
Comment (sbc100) [inline src/shell.html:193]: Can we 80 columns to avoid adding quite so many lines?

Comment (yecril71pl) [reply src/shell.html:193]: We can, I do not care, do you?

Comment (sbc100) [reply src/shell.html:193]: I would rather leave at as a single huge line than add thousands to tiny lines to this file.    I'm actually not too worried about a couple of huge lines.  My editor doesn't seem to mind to much.

Also, this data is the direct output of tool that generates svg file so inserting it unmodified/unformatting into the html seems like a reasonable approach maybe?

Comment (sbc100) [reply src/shell.html:193]: In other words, I would rather not add any more lines that needed here.

Comment (sbc100) [reply src/shell.html:193]: Perhaps you can explain a little more about the issues you are having with your editor and these lines?

Comment (yecril71pl) [reply src/shell.html:193]: If the SVG is automatically generated, you must reveal how it is generated, otherwise this file is not open source (because it is not really a pristine source).

Comment (sbc100) [reply src/shell.html:193]: I believe the embedded file here is `emscripten_powered_by_logo.svg` from the docs directory: https://github.com/emscripten-core/emscripten/tree/main/docs.  

It was contributed back in 857458f5e889b5d3b94fee2cfed2f8eb3e27d9c2.

Comment (yecril71pl) [reply src/shell.html:193]: I still cannot see how it was generated.  Which is, by itself, OK, as long as there is no process to generate it, it is a pristine source as a work of art.
Inserting an `SVG` file into an `HTML` file, by itself, is OK because a HTML file can be served as `application/html`.  Inserting it as `application/svg-xml`, however, rather than as `text/html`, is incompatible with the suggestion that the end user might want to edit the result, assuming that we do not preclude editing it in a text editor, and I do not think it would be a good move to forbid that, considering that a structural editor is unlikely to complain when presented a `text/html` version of the file.  It may, of course, save it as `application/html` after editing, but it is not our problem any more.  
TL;DR: The original artwork must be converted to `text/html` before inserting.
